.env

```diff
@@ -20,6 +20,7 @@ KEYCLOAK_DB_IMAGE_TAG=latest
 ESANJEEVANI_SERVICE_IMAGE_TAG=latest
 METABASE_IMAGE_TAG=latest
 METABASE_DB_IMAGE_TAG=latest
+AVNI_WEBAPP_IMAGE_TAG=latest
```

Comment:
Can we group all the images tags from avniproject under a separate comment so that it would be easy to track ?

Reply:
done